packages/mask-sdk/public-api/mask-wallet.ts

```diff
@@ -82,15 +255,15 @@ declare namespace Mask.Ethereum {
         accountsChanged: CustomEvent<string[]>
     }
     export interface MaskEthereumEventEmitter extends EthereumEventEmitter, EventTarget {
-        addEventListener<K extends keyof EthereumEventMap>(
-            type: K,
-            callback: EventListenerOrEventListenerObject | null | ((ev: EthereumEventMap[K]) => any),
-            options?: boolean | AddEventListenerOptions,
-        ): void
-        removeEventListener<K extends keyof EthereumEventMap>(
-            type: K,
-            listener: EventListenerOrEventListenerObject | null | ((ev: EthereumEventMap[K]) => any),
-            options?: boolean | EventListenerOptions,
-        ): void
+        // addEventListener<K extends keyof EthereumEventMap>(
+        //     type: K,
+        //     callback: EventListenerOrEventListenerObject | null | ((ev: EthereumEventMap[K]) => any),
+        //     options?: boolean | AddEventListenerOptions,
+        // ): void
+        // removeEventListener<K extends keyof EthereumEventMap>(
+        //     type: K,
+        //     listener: EventListenerOrEventListenerObject | null | ((ev: EthereumEventMap[K]) => any),
+        //     options?: boolean | EventListenerOptions,
+        // ): void
```

Comment:
```suggestion
```

Reply:
those APIs are intended to be implemented in the future so I'd like to keep them.